src/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContext.cs

```diff
@@ -88,112 +88,93 @@ public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly)
                 }
 
                 Uri licenseFile = null;
-
+                // the AppDomain.CurrentDomain.SetupInformation.LicenseFile always returns null.
+                // This means we have to find the license file using the fallback logic below.
```

Comment:
Do you want to add an assert for this, in case it changes in the future and this code needs to come back?